app/controllers/provider_interface/offer/conditions_controller.rb

```diff
@@ -38,8 +38,7 @@ def update
     private
 
       def conditions_params
-        params.require(:provider_interface_offer_wizard).permit(:further_condition_1, :further_condition_2,
-                                                                :further_condition_3, :further_condition_4,
+        params.require(:provider_interface_offer_wizard).permit(further_conditions: {},
```

Comment:
Not sure if there's a way to be more strict about the params here, since we don't necessarily know the keys of the hash

Reply:
I think its ok to accept that we have dynamic params in `further_conditions`